api/HTMLDataListElement.json

```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
+    "HTMLDataListElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLDataListElement",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "62"
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22"
```

Comment:
Firefox 22 is not in the original table, right? 
I would have expected this to be like https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist#Browser_compatibility, but I would be okay with this being just `true` for now, too.